msp/msp_exp_handler.c

```diff
@@ -738,6 +734,22 @@ void msp_exprecv_syscommand(unsigned char opcode)
 }
 
 
+
+/**
+ * @brief Should be called at completion of a MSP command.
+ * Accumulates a counter and saves sequence flags to NVM at an interval defined
+ * by SEQ_FLAG_SAVE_INTERVAL
+ */
+void accumulate_msp_seq_flag_save(void)
```

Comment:
This function should be made `static` (so it's file-local) and moved to the top of the file, for example right below `prep_payload_data()`. Without the move to the top of the file, there are warnings for undefined reference to this function -- and the possibility for crashes when the code is run.

It also needs to be called as part of `msp_exprecv_syscommand()`.